lib/airbrake/rails3_tasks.rb

```diff
@@ -59,6 +59,11 @@ def wait_for_threads
     class ActionDispatch::DebugExceptions; def call(env); @app.call(env); end; end
     class ActionDispatch::ShowExceptions; def call(env); @app.call(env); end; end
 
+    # Override BetterErrors middleware if present
```

Comment:
How it is possible that there can be several copies of `BetterErrors` middleware in airbrake namespace? Probably I don't understand how Ruby/Rails works.